helm/hpcc/values.yaml

```diff
@@ -365,48 +383,59 @@ esp:
   application: eclwatch
   auth: none
   replicas: 1
-  ## port can be used to change the local port used by the pod. If omitted, the default port (8880) is used
-  port: 8888
-  ## servicePort controls the port that this service will be exposed on, either internally to the cluster, or externally
-  servicePort: 8010
-  ## Specify public: true if you want the service available from outside the cluster. Typically, eclwatch and wsecl are published externally, while eclservices is designed for internal use.
-  public: true
+  service:
+    ## port can be used to change the local port used by the pod. If omitted, the default port (8880) is used
+    port: 8888
+    ## servicePort controls the port that this service will be exposed on, either internally to the cluster, or externally
+    servicePort: 8010
+    ## Specify visibility: local (or global) if you want the service available from outside the cluster. Typically, eclwatch and wsecl are published externally, while eclservices is designed for internal use.
+    visibility: local
+    ## Annotations can be specified on a service - for example to specify provider-specific information such as service.beta.kubernetes.io/azure-load-balancer-internal-subnet
+    annotations:
+      service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "mysubnet"
```

Comment:
comment out? or remove entirely now you have your visibility section?  Same for the label below.

Reply:
I should comment it out. The idea is that you can override on individual services as well as setting up visibility "classes" of service.